src/test/java/org/joda/time/tz/TestZoneInfoCompiler.java

```diff
@@ -0,0 +1,80 @@
+package org.joda.time.tz;
```

Comment:
Can you add the same license header as other files in the repo. In doing do, you are granting copyright to me to allow the project to be released as a whole under a single license.